util/conditions/sort.go

```diff
@@ -94,42 +94,42 @@ func defaultSortLessFunc(i, j metav1.Condition) bool {
 // | Deleting                       | x       | x   | x  | x  | x  | x       |
 // .
 var order = []string{
-	clusterv1.AvailableV1Beta2Condition,
-	clusterv1.ReadyV1Beta2Condition,
-	clusterv1.MachineUpToDateV1Beta2Condition,
-	clusterv1.ClusterRemoteConnectionProbeV1Beta2Condition,
-	clusterv1.BootstrapConfigReadyV1Beta2Condition,
-	clusterv1.InfrastructureReadyV1Beta2Condition,
-	clusterv1.ClusterControlPlaneInitializedV1Beta2Condition,
-	clusterv1.ClusterControlPlaneAvailableV1Beta2Condition,
-	clusterv1.ClusterWorkersAvailableV1Beta2Condition,
+	clusterv1.AvailableCondition,
+	clusterv1.ReadyCondition,
+	clusterv1.MachineUpToDateCondition,
+	clusterv1.ClusterRemoteConnectionProbeCondition,
+	clusterv1.BootstrapConfigReadyCondition,
+	clusterv1.InfrastructureReadyCondition,
+	clusterv1.ClusterControlPlaneInitializedCondition,
+	clusterv1.ClusterControlPlaneAvailableCondition,
+	clusterv1.ClusterWorkersAvailableCondition,
 	kubeadmControlPlaneCertificatesAvailableV1Beta2Condition,
```

Comment:
```suggestion
	kubeadmControlPlaneCertificatesAvailableCondition,
```
Same for all the conditions we are copying from KCP